docs/python/binary-extension.rst

```diff
@@ -49,25 +49,59 @@ The command line tool `auditwheel`_ adds a ``bezier/.libs`` directory
 with a modified ``libbezier`` and all of its dependencies (e.g.
 ``libgfortran``)
 
-.. code-block:: console
+.. testsetup:: linux-libs, linux-readelf-py, linux-readelf-lib, macos-dylibs,
+               macos-extension, macos-delocated-libgfortran
+
+   import os
+   import subprocess
+
+   import bezier
+   import tests.utils
+
+
+   print_tree = tests.utils.print_tree
+   base_dir = os.path.abspath(os.path.dirname(bezier.__file__))
+   # macOS specific.
+   dylibs_directory = os.path.join(base_dir, ".dylibs")
+   # Linux specific.
+   libs_directory = os.path.join(base_dir, ".libs")
 
-   $ cd .../site-packages/bezier
-   $ ls -1 -F .libs/
-   libbezier-4f59b4c5.so.2020.1.14*
```

Comment:
Sheesh! It's good I did this, this hash (`4f59b4c5`) differs from the other hash (`28a97ca3`) in the `readelf` bits.